src/trustyai/utils/server/server.py

```diff
@@ -0,0 +1,29 @@
+"""
+Server module
+"""
+# pylint: disable = import-error, too-few-public-methods
+from kubernetes import config, client
+from openshift.dynamic import DynamicClient
+
+
+class TrustyAIServer:
+    """
+    Gets TrustyAI services information
+    """
+
+    def __init__(self):
+        k8s_client = config.new_client_from_config()
```

Comment:
`new_client_from_config()` is actually calling `load_kube_config()`, which reads the context locally. This will work from outside the cluster, but probably not from inside.

On the other hand, `load_incluster_config()` uses the pod's service account. Works from inside the cluster, but not from outside.

Perhaps we could have something like (untested):

```python
try:
        k8s_client = config.load_incluster_config()
except config.ConfigException:
         k8s_client = config.load_kube_config()
```